File: samples/JavaScript/Json Store/Cursor/Select/By UNID.js
session.setAsync(false);

var store = session.getDatabase("playground").getStore("pinball");
var s = "";

s += ">>> Select a document by UNID\n";
var c = store.openCursor().range(0,5);

// Select by UNID
c.unid("1000").find(function(e) {
  s += "  value:"+darwino.Utils.toJson(e.getValue(),false)+"\n";
});  
s += "\n";


darwino.Utils.setText("content","{0}",s);

File: samples/Java/Json Store/Cursor/Select/By UNID.java
Store store = session.getDatabase("playground").getStore("pinball");
String s = ""; final StringBuilder b = new StringBuilder();

s += ">>> Select a document by UNID\n";
Cursor c = store.openCursor().range(0,5);
b.setLength(0);
c.unid("1000").find(new CursorHandler() {
  public boolean handle(CursorEntry e) throws JsonException {  
  	b.append("  value:"+e.getValue()+"\n");
    return true;
  }
});  
s += b.toString()+"\n";


_formatText("{0}",s);